File: AndroidOCourse/InteractivityDemo/app/src/main/java/com/example/kdotz/interactivitydemo/MainActivity.java
package com.example.kdotz.interactivitydemo;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.EditText;
import android.widget.ImageView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    boolean flag = false;

    public void clickFunction(View view) {

        ImageView image = (ImageView) findViewById(R.id.imageView);
        Log.i("Info", "Button Pressed!");
        switch (view.getId()) {
            case R.id.button2: {
                if (flag) {
                    image.setImageResource(R.drawable.unicorn);
                    flag = false;
                } else {
                    image.setImageResource(R.drawable.unicorn2);
                    flag = true;
                }
                return;
            }
        }
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: AndroidOCourse/PhotoImportDemo/app/src/main/java/com/kdotz/photoimportdemo/MainActivity.kt
package com.kdotz.photoimportdemo

import android.content.Intent
import android.os.Bundle
import android.provider.MediaStore
import android.support.v7.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        Intent intent = new Intent(Intent.ACTION_PICK, MediaStore.Images.Media.EXTERNAL_CONTENT_URI);



    }
}
